content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -2,148 +2,105 @@
 Getting started
 ===============
 
-When working with a Point of Sale application, employees want a
-simple, and user-friendly solution. A solution that works online or
-offline and with any device.
+**Point of Sale** offers employees a simple and user-friendly solution that works online or offline
+with any device.
 
-A Point of Sale system is a fully integrated application that allows any transaction, 
-automatically registers product moves in your stock, and gives you
-real-time statistics and consolidations across all shops.
+A point of sale system is a fully integrated application that allows any transaction, automatically
+registers product moves in your stock, and gives you real-time statistics and consolidations across
+all shops.
 
 Configuration
 =============
 
-Make products available in the PoS
-----------------------------------
+Make products available
+-----------------------
 
-To make products available for sale, go to :menuselection:`Point of
-Sale --> Products --> Products` and open a product. In the *Sales*
-tab, enable *Available in Point of Sale*.
+To make products available for sale, go to :menuselection:`Products --> Products`, and open a
+product. In the :guilabel:`Point of Sale` tab, enable :guilabel:`Available in POS`.
 
-.. image:: getting_started/getting_started_01.png
-    :align: center
-    :alt: Product form. Making the product available in PoS.
-
-.. note::
-   You can also define if the product has to be weighted or not.
+.. image:: getting_started/pos-available.png
+   :align: center
+   :alt: Making a product available in PoS.
 
 Configure your payment methods
 ------------------------------
 
-To add a payment method, you first need to create it. Go to
-:menuselection:`Point of Sale --> Configuration --> Payment Methods`
-and click on create. Then, you can name your payment method and set it
-up.
+To add a payment method, you first need to create it. Go to :menuselection:`Configuration -->
+Payment Methods -->Create`. Name your payment method and select the **payment terminal** or check
```

Comment:
```suggestion
Payment Methods --> Create`. Name your payment method and select the :guilabel:`payment terminal` for card transactions, or check
```
Not sure about guilabels, but since it is a visible field, maybe? Also, I think it might be worth mentioning that you need to enter credentials for the terminal payments